packages/types/src/saml.ts

```diff
@@ -17,7 +17,7 @@ export const SAML_IDPS: SamlIdpMap = {
     logo: 'google',
   },
   saml_microsoft: {
-    name: 'Microsoft Azure AD',
+    name: 'Microsoft Entra ID (Formerly AD)',
```

Comment:
🙃
```suggestion
    name: 'Microsoft Entra ID (Azure AD)',
```

Reply:
I tend to prefer `Formerly` since it denotes that AD is deprecated.